src/js/tracks/audio-track-list.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file audio-track-list.js
+ */
+import TrackList from './track-list';
+
+/**
+ * A list of possiblee audio tracks. All functionality is in the
+ * base class Tracklist and the spec for AudioTrackList is located at:
+ * @link https://html.spec.whatwg.org/multipage/embedded-content.html#audiotracklist
+ *
+ * interface AudioTrackList : EventTarget {
+ *   readonly attribute unsigned long length;
+ *   getter AudioTrack (unsigned long index);
+ *   AudioTrack? getTrackById(DOMString id);
+ *
+ *   attribute EventHandler onchange;
+ *   attribute EventHandler onaddtrack;
+ *   attribute EventHandler onremovetrack;
+ * };
+ *
+ * @param {AudioTrack[]} tracks a list of audio tracks to instantiate the list with
+ # @extends TrackList
+ * @class AudioTrackList
+ */
+class AudioTrackList extends TrackList {
+  addTrack_(track) {
+    super(track);
+    track.addEventListener('enabledchange', () => {
+      this.trigger('change');
+    });
+  }
+  addTrack(track) {
```

Comment:
We need a way for users of videojs to add tracks to the track lists, the lists are read-only and done by the "vendor". I don't think projects like contrib-hls should be using a "private" function like `addTrack_` so, I feel like renaming it to `addTrack` is best. Also, I think this is the time to probably bring in a stability index like https://nodejs.org/dist/latest-v5.x/docs/api/documentation.html because the public functions could go away one MSE full supports these natively in which case we could pick the additional tracks that way.